Resources/Documentation/troubleshooting.md

```diff
@@ -56,10 +56,6 @@ Error while decoding stream #0:0: Invalid data found when processing input
 
 You can safely ignore this warning, it's just FFmpeg complaining that it didn't receive a full frame of video when vrecord stopped.
 
-#### Sync Issues with FFmpeg 4.0 and qcli
```

Comment:
There is a similar statement in settings.md, if removed, I suggest they be removed together.

Reply:
Should a line instead be included to explain why the option exists? i.e., it is recommended to create *during* capture but the option to create after can be used to troubleshoot. I can see a user being confused as to how/why they would pick one over the other.

Reply:
Also, the warning in settings.md was not mine, so I was trying to just get back to structural revision, dealing with content subsequently as you suggested

Reply:
Replacing the warning with an explanation sounds fine to me, whether in this pr or a subsequent one.

Reply:
I'll just remove the settings.md warning for now for consistency and to resolve #333 , can move feature clarification/further content audit into a subsequent PR